Integrations/integration-VirusTotal.yml

```diff
@@ -421,16 +445,16 @@ script:
 
     switch (command) {
         case 'test-module':
-            doIP('8.8.8.8'); // Check IP 8.8.8.8 - it will throw an error if not successful
+            doFile('7657fcb7d772448a6d8504e4b20168b8'); // Check sample file - it will throw an error if not successful
```

Comment:
Why? This is why we have defaults in the code as well. Also, did not provide the other things like threshold.